docs/up-logs/downloads/v1.md

```diff
@@ -60,10 +60,15 @@ For Example:
             <td>34</td>
             <td>1</td>
         </tr>
+        <tr>
```

Comment:
I'm not sure if you want to remove the up-logs additions? I think they are still accurate so you probably don't need to :thinking:

Reply:
I'll delete the extra row as the up-ga ones for consistency